docs-next/pages/apis/fields.mdx

```diff
@@ -74,12 +76,16 @@ export default config({
     ListName: list({
       fields: {
         fieldName: text({
-          access: { ... },
-          hooks: { ... },
-          label: '...',
+          access: {
+            /* ... */
+          },
+          hooks: {
+            /* ... */
+          },
+          label: '/* ... */',
           ui: {
-            views: '...',
-            description: '...',
+            views: '/* ... */',
```

Comment:
```suggestion
            views: '...',
```
For string values can we keep it just as `...`? I'd like to keep as much visual noise out of the way as possible, so that readers can focus on the meat of the documentation.